packages/application-shell/src/components/theme-switcher/messages.ts

```diff
@@ -0,0 +1,13 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
```

Comment:
Please remember to run the `pnpm extract-intl` script to generate the messages in the `core.json` file.

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3445/commits/2d35ba79fb6e905fef1a1c40f231e12aa9328816

Thank you